### DISCUSSION TITLE
image file links in orgzly-revived are different to file links expected by emacs on PC - how do I resolve this?

### DISCUSSION THREAD

**Discussion by superhans2** [orgzly-revived/orgzly-android-revived#421] [General]
I have a directory `~/org/` on my PC which contains a subfolder `data` which contains images I am displaying in the org files. 
I sync this directory with my phone using orgzly-revived.

There is a difference in how the filepath is searched by orgzly-revived:
If I try using relative filepath like `./data/image-example.png` (which works on my PC), on orgzly it expands to `/storage/emulated/0/data/image-example.png` and is not found. If I set the path to `org/data/image-example.png` it finds the image and works on orgzly but on my PC it is looking in `org/org/data/image-example.png` and is not found. 

It seems the filepaths are relative to the org directory in orgzly, but relative to the org files on my PC. I'm not sure how to work around this?

I have also tried using absolute filepath but this also doesn't work: if I reference an image in the data directory as `[[~/org/data/image-example.png]]` the filepath expands to `/storage/emulated/0/~/org/data/image-example.png` and is not found.

Many thanks

**Comment by amberin** [comment]
Hi! Have you tried changing the following settings?

Notes & notebooks -> Root for absolute links
Notes & notebooks -> Root for relative links

**Comment by superhans2** [comment]
I didn't know about these settings, thanks for pointing me to them, I'll have a poke about see if I find something that works xx
